eventsources/eventing.go

```diff
@@ -223,6 +224,13 @@ func GetEventingServers(eventSource *v1alpha1.EventSource) map[apicommon.EventSo
 		}
 		result[apicommon.PulsarEvent] = servers
 	}
+	if len(eventSource.Spec.Generic) != 0 {
```

Comment:
Is `Generic` `Recreat` or `RollingUpdate` strategy? If it's `recreate`, need to add it in the slice in pkg/apis/common/common.go

Reply:
Done.

Reply:
done